docs/DeveloperGuide.md

```diff
@@ -4,9 +4,123 @@
 
 {list here sources of all reused/adapted ideas, code, documentation, and third-party libraries -- include links to the original source as well}
 
-## Design & implementation
+## Design & implementation, Architecture
 
 {Describe the design and implementation of the product. Use UML diagrams and short code snippets where applicable.}
```

Comment:
It would be better to understand if there was a diagram for what you implemented.